src/locales/en_US.js

```diff
@@ -828,15 +828,10 @@ const strings = {
 
   // Account deletion:
   delete_account_title: 'Delete Account',
-  delete_account_funds:
-    'Deleting your Edge account will make any funds in your wallets inaccessible. Any future funds that these wallets may receive will also be inaccessible.',
-  delete_account_partner:
-    'Accounts created with Edge buy / sell partners will remain active. You will need to request deletion for any such accounts directly with the appropriate partners.',
-  delete_account_devices:
-    'If this account has logged into any other devices, the private keys and information on those devices will remain accessible until the application has been deleted from those devices.',
-  delete_account_username: "Deletion of this account will make the account's username usable for new account creation.",
-  delete_account_support: 'If you have any questions or concerns, feel free to contact us, or email support@edge.app prior to deleting your account.',
-
+  delete_account_body:
+    "Deleting your Edge account will make any funds in your wallets inaccessible. Any future funds that these wallets may receive will also be inaccessible.\n\nAccounts created with Edge buy / sell partners will remain active. You will need to request deletion for any such accounts directly with the appropriate partners.\n\nIf this account has logged into any other devices, the private keys and information on those devices will remain accessible until the application has been deleted from those devices.\n\nDeletion of this account will make the account's username usable for new account creation.\n\nIf you have any questions or concerns, feel free to contact us, or email support@edge.app prior to deleting your account.",
```

Comment:
"Deleting your Edge account will make any funds in your wallets inaccessible. Any future funds that these wallets may receive will also be inaccessible.\n\nAccounts created with Edge buy / sell partners will remain active. You will need to request deletion for any such accounts directly with the appropriate partners.\n\nIf this account has logged into any other devices, the private keys and information on those devices will remain accessible until the application has been deleted from those devices.\n\nDeletion of this account will make the account's username usable for new account creation.\n\nIf you have any questions or concerns, please contact us at support@edge.app prior to deleting your account."